web/src/admin.js

```diff
@@ -199,6 +199,14 @@ export function build_page() {
         realm_enable_read_receipts: page_params.realm_enable_read_receipts,
         allow_sorting_deactivated_users_list_by_email:
             settings_users.allow_sorting_deactivated_users_list_by_email(),
+        roles: [
+            {role: $t({defaultMessage: "All roles"}), code: 0},
+            {role: $t({defaultMessage: "Owners"}), code: 100},
```

Comment:
Changed it to use `setttings_config.user_role_values` to list all the roles, however we need the dropdown to list roles in a hierarchical order from higher level to lower level. Therefore,reversed the return value of the function before appending it. Additionally, we wanted to include the title `All roles` in the dropdown, so hardcoded it since it is
not present in `user_role_values`.